CHANGELOG.md

```diff
@@ -22,6 +22,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 - Refactor CAN to use the [`bxCan`](https://github.com/stm32-rs/bxcan) crate. ([#207])
 - Implement `Serial::join` which allows to re-create the serial peripheral,
   when `Serial::split` was previously called. ([#252])
+- Parameterized `usb::Peripheral` and `usb::UsbType` on the pin configuration
+  used ([#255])
```

Comment:
There is still 

```
[#255]: https://github.com/stm32-rs/stm32f3xx-hal/pull/255
```

missing at the end of the file, to make this link work. (Line 350 onwards) 

Sorry :see_no_evil: I probably should add a comment to the changelog to make more obvious.